### PR TITLE
Fix failures on displaying getters

### PR DESCRIPTION
**Pull request by annagrin**
Add evaluation on instance IDs (in addition to library ids).

Closes: https://github.com/dart-lang/webdev/issues/2309
Closes: https://github.com/dart-lang/webdev/issues/2351

### CODE REVIEW

**Review by elliette — APPROVED** [review]

File: dwds/test/evaluate_common.dart [hunk 1]
```diff
@@ -699,6 +699,28 @@ void testAll({
...
+          'RecordType getters',
+          () async {
+            final libraryId = getRootLibraryId();
+
+            final type = await getInstanceRef(libraryId, '(0,1).runtimeType');
+            final result = await getInstanceRef(type.id, 'hashCode');
+
+            expect(result, matchInstanceRefKind('Double'));
+          },
+          skip: 'https://github.com/dart-lang/sdk/issues/54609',
```
Comment (elliette) [inline dwds/test/evaluate_common.dart:712]: Can this be merged with https://github.com/dart-lang/webdev/issues/2351 or are they separate issues?

File: dwds/CHANGELOG.md [hunk 2]
```diff
@@ -6,6 +6,7 @@
 
 - Send untruncated `dart:developer` logs to debugging clients. - [#2333](https://github.com/dart-lang/webdev/pull/2333)
 - Enabling tests that run with the DDC module system and exposing `utilities/ddc_names.dart` - [#2295](https://github.com/dart-lang/webdev/pull/2295)
+- Fix errors on displaying getters. - [#2343](https://github.com/dart-lang/webdev/pull/2343)
```
Comment (elliette) [inline dwds/CHANGELOG.md:9]: Nit: specify this is only for VS Code

**Review by elliette — APPROVED** [review]

File: dwds/test/evaluate_common.dart (hunk 1 above)
Comment (annagrin) [reply dwds/test/evaluate_common.dart:712]: Yes, I will mark this change as fixing https://github.com/dart-lang/webdev/issues/2351 as well. My change effectively fixes the issue that caused https://github.com/dart-lang/webdev/issues/2351

File: dwds/CHANGELOG.md (hunk 2 above)
Comment (annagrin) [reply dwds/CHANGELOG.md:9]: I think this is for anything that uses sdk dap - could be devtools in the future?

**Pull request merged**